VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -97,6 +99,8 @@ public void notifyNetworkConnect(){
         }
     }
 
+    // This flag is to enable the login and sign-up Activities to be
+    // excluded from the download progress update callback.
     protected boolean runOnTick = true;
```

Comment:
The `onTick()` callback is meant to be generic and overridable by subclasses for their own purposes. However, it seems that it's only usage is to keep the progress bar menu item up to date. I'll unbundle that menu to the relevant Activities and restructure this logic when resolving MA-1665.